components/ui/tooltip.tsx

```diff
@@ -20,7 +20,7 @@ const TooltipContent = React.forwardRef<
       ref={ref}
       sideOffset={sideOffset}
       className={cn(
-        "z-50 overflow-hidden min-w-48 rounded-md bg-neutral-50 px-3 py-1.5 text-xs animate-in fade-in-0 zoom-in-95 data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=closed]:zoom-out-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2",
+        "TooltipContent z-50 overflow-hidden rounded-md bg-neutral-50 px-3 py-1.5 text-lg",
```

Comment:
Sorry this is a nit-pick, can the `TooltipContent` class name be in kabob-case like the other class names?

Reply:
Actually didn't even need that! I removed it. It's for if we have custom css styles for defined for tooltip content.